src/components/post-list/Post-list.vue

```diff
@@ -5,7 +5,7 @@
     <li v-for="post in posts">
       <Post
         :title="post.title"
-        :description="post.description"
+        :summary="post.summary"
         :votes="post.votes"
```

Comment:
Description is better I think

Reply:
Nice one. Confirmed with Mark, no longer relevant.